packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -17,6 +17,13 @@
 					<div class="toolbar">
 						<p>Click Run to start optimization.</p>
 						<span class="flex gap-2">
+							<Button
+								label="Reset"
+								outlined
+								@click="resetState"
+								severity="secondary"
+								:disabled="_.isEmpty(node.outputs[0].value)"
```

Comment:
I think I will have to make some complicated logic to make this work (similar to intervention policy and model config).  The knobs does not necessarily equal state so I will have to look into this a bit more.  It might be worth it to merge this and I can look at this later.